designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ManagementResources.scala

```diff
@@ -69,7 +69,7 @@ object ManagementResources {
         "variables" -> a.variables.asJson
       )
 
-    implicit val componentInfoEncoder: Encoder[ComponentInfo]         = deriveConfiguredEncoder
+    implicit val componentInfoEncoder: Encoder[ComponentId]           = deriveConfiguredEncoder
```

Comment:
```suggestion
    implicit val componentIdEncoder: Encoder[ComponentId]           = deriveConfiguredEncoder
```

Reply:
Fixed, in fact this encoder wasn't needed here :)